tests/test_type_inference/test_attribute.py

```diff
@@ -12,12 +12,12 @@ def test_instance_dot_method():
         class A:
             def foo(self, x):
                 return x + 1
-                
+
         A().foo(0)
         '''
     module, _ = cs._parse_text(program, reset=True)
     for attribute_node in module.nodes_of_class(astroid.Attribute):
-        assert attribute_node.inf_type.getValue() == Callable[[int], int]
+        assert str(attribute_node.inf_type.getValue()) == "typing.Callable[[int], int]"
```

Comment:
Why was this change necessary?

Reply:
Direct comparison between types doesn't always work as expected. That's why we compare strings and not types in `test_tnode_structure` and also why we use strings in the lookup dictionaries. These tests started failing for no apparent reason, so I did the same thing here.